packages/ckeditor5-package-generator/lib/templates/ts/typings/ckeditor5-inspector.d.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * @license Copyright (c) 2022-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+declare module '@ckeditor/ckeditor5-inspector' {
+	const inspector: {
+		attach( editor: any ): void; /* eslint-disable-line no-unused-vars */
```

Comment:
This `eslint` rule seems to be not needed. I don't see any linter errors after removing it.